modules/cli-authentication.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cli/using-roxctl-cli.adoc
+:_module-type: PROCEDURE
+[id="cli-authentication_{context}"]
+= Authenticating using the roxctl CLI
+
+For authentication, you can use an authentication token or your administrator password.
+Red Hat recommends using an authentication token in a production environment because each token is assigned specific access control permissions.
+//TODO: Add links to role based access control
+
+Use the following steps to generate an authentication token.
+
+.Procedure
+
+. Navigate to the RHACS portal.
+. Go to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Authentication Tokens* category, and click *API Token*.
+. Click *Generate Token*.
+. Enter a name for the token and select a role that provides the required level of access (for example, *Continuous Integration* or *Sensor Creator*).
+. Click *Generate*.
++
+[IMPORTANT]
+====
+Copy the generated token and securely store it.
+You will not be able to view it again.
+====
+
+[NOTE]
+====
+After you have generated the authentication token, export it as `ROX_API_TOKEN` variable:
```

Comment:
It's fine if it was this way before, but just wondering if this step was something that they have to do? If so, I'd consider putting it in the above procedure instead of in a note below.

Reply:
It's not a requirement but a recommendation and that's why I kept it out from the procedure.